package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chartmogul-node",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
[suggestion]
```suggestion
  "version": "1.3.0",
```

Reply:
You're right. I've corrected the version to "1.3.0".